### ISSUE TITLE
Confused: Is there a COPR for LLL? (copr.fedorainfracloud.org)

### ISSUE THREAD

**Issue by msmafra** [johnfanv2/LenovoLegionLinux#146]
Hello there

Is there a COPR available to install LLL?
I saw that is described as PPA instead of COPR, but did not find an URL nor found in copr.fedorainfracloud.org. 
Are those just to generate RPMS?

**Comment by MrDuartePT** [comment]
Hello.

I dont use the Fedora COPR and I not planing on using since I don't use fedora as my main OS.
If you what to maintain the package on COPR pls tag me, also in the repo you have the workflow and the script on how to build the package.

Pls use the project fedora repo (`inside the package_repo folder`) to install the package!

**Comment by MrDuartePT** [comment]
On a second thought I could try to use Copr since it only require to provide the spec file if work well I will just drop the repo here and use corp instead

**Comment by msmafra** [comment]
I dabbled in COPR spec files once many years back for two packages I wanted to use on Fedora, but I gave up. I also read and article/post (don´t recall the url) about the quirks and documentation problems/inconsistency of RPMs sometime back. I don't know if that got improved.
Since I use CachyOS's kernel as my default on Fedora nowadays, LLL is already integrated by them. The COPR would be a huge help when using Fedora's  kernel.
